nuxt.config.js

```diff
@@ -172,20 +172,12 @@ export default {
     ],
     lazy: true,
     langDir: 'locales',
-    strategy: 'no_prefix',
     defaultLocale: 'en',
     /**
      * This section is critical for the current, iframed production environment
      * {@link https://i18n.nuxtjs.org/options-reference/#detectbrowserlanguage}
      * */
-    detectBrowserLanguage: {
-      useCookie: true,
-      cookieKey: 'i18n_redirected',
-      alwaysRedirect: true,
-      cookieCrossOrigin: true,
-      cookieSecure: true,
-    },
-    baseUrl: 'http://localhost:8443',
+    detectBrowserLanguage: false,
```

Comment:
Now we'd only look at the path supplied to the iframe, no other communication necessary.

Reply:
@zackkrida can you update the comment for this variable? If it's still critical, we should describe why and how, especially now that it's simply disabled.

Reply:
Thanks for the suggestion, I updated the comment with more useful information!